.github/workflows/Tests.yml

```
@@ -166,15 +166,15 @@ jobs:
     timeout-minutes: 10
 
     env:
-      PYTHON_VERSION: '3.13'
+      PYTHON_VERSION: "${{ vars.PYTHON_DEFAULT }}"
       LANG: "en_US.utf-8"
 
     steps:
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)